CIRC-8497: Update CKA to support k8s versions v1.20+

aesca1er opened the pull request.

dhaifley reviewed (approved):
The DNS changes all LGTM. I guess I really don't understand all of the changelog stuff. Why are these large (9434 line files) additions to the repo needed? I still don't understand why if I wanted to see a list of commit messages I wouldn't just pull up a compare between releases in GitHub? What do we gain by keeping this info in large files like this in the repo?

aesca1er commented:
> The DNS changes all LGTM. I guess I really don't understand all of the changelog stuff. Why are these large (9434 line files) additions to the repo needed? I still don't understand why if I wanted to see a list of commit messages I wouldn't just pull up a compare between releases in GitHub? What do we gain by keeping this info in large files like this in the repo?

That file is used by goreleaser's `chglog` changelog generator.
It isn't really necessary to keep it in vcs as it can be generated as required.
I added the file to the .gitignore and removed it.
